### PR TITLE
Add application presenter

### PR DESCRIPTION
**Pull request by willmcb**
## Description of change
Adds presenter and spec for Applications + amends status copy to sentence case

## Link to relevant ticket
[CRIMAP-116](https://dsdmoj.atlassian.net/browse/CRIMAP-116)

## How to manually test the feature
load the `/applications` page it should look the same as it did previously

status should be in lower case

### CODE REVIEW

File: config/locales/en/shared.yml [hunk 1]
```diff
@@ -2,5 +2,5 @@ en:
   shared:
```
Comment (zheileman) [inline config/locales/en/shared.yml:2]: Not sure we will be reusing this in other places. Might be more clean to keep it all together for now in the `dashboard.yml` index section? Perhaps `crime_applications.index.status.{xxx}`
We have also in there the table headings so kind of flows nice.

File: spec/presenters/application_presenter_spec.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationPresenter do
+  include ApplicationHelper
```
Comment (zheileman) [inline spec/presenters/application_presenter_spec.rb:4]: So we don't have to include this (as it is really not the concern of this presenter) we can always use this way to instantiate the presenter:

`described_class.new(crime_application)`

Maybe cleaner?

File: spec/presenters/application_presenter_spec.rb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+  let(:presented_application) {
```
Comment (zheileman) [inline spec/presenters/application_presenter_spec.rb:6]: Would this be the subject under test?

subject { ... }

**Review by zheileman — COMMENTED** [review]
I added a few suggestions!
Nothing really blocking 🚀

File: config/locales/en/shared.yml (hunk 1 above)
Comment (willmcb) [reply config/locales/en/shared.yml:2]: Makes sense I will amend 👍

**Review by zheileman — APPROVED** [review]
🥳

File: app/controllers/crime_applications_controller.rb [hunk 4]
```diff
@@ -35,6 +35,8 @@ def confirm_destroy; end
   private
 
   def set_applicant_name
-    @applicant_name = current_crime_application.applicant.full_name
+    @applicant_name = CrimeApplicationPresenter
```
[the remark this answers is not in the document]
Comment (willmcb) [reply app/controllers/crime_applications_controller.rb]: Seems to work okay

**Pull request merged**